### PR TITLE
account page styles

### PR DESCRIPTION
**Pull request by stormcloud266**
The new account page styles

Screenshot 2024-09-11 at 5 13 17 PM

### CODE REVIEW

**Comment by stormcloud266** [comment]
* **#711** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/711?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]
Light theme looks off. You'll have to manually set all the green to black, since the color themes don't support this yet.

`$theme-light:{{col: "$color12"}}`

Everything else looks good

**Comment by stormcloud266** [comment]
@youngkidwarrior tried olive on the icons, let me know what you think. also fixed that stretch issue on tall screens

Screenshot 2024-09-12 at 5 49 09 PM

File: packages/app/features/account/screen.tsx [hunk 1]
```diff
@@ -75,346 +72,292 @@ export function AccountScreen() {
...
+      $theme-light={{ hoverStyle: { borderColor: '$olive' } }}
     >
-      <XStack ai={'center'} gap={'$size.0.75'} mb="$size.0.75">
-        <Heading fontSize={'$9'} fontWeight={'600'} color={'$color12'} textTransform="uppercase">
-          {title}
-        </Heading>
-        {title === 'Sendtags' && <InfoDialog />}
+      <XStack jc={'space-between'} ai={'center'}>
+        <XStack gap={'$size.0.9'} ai={'center'}>
+          <Button.Icon>{icon}</Button.Icon>
```
Comment (youngkidwarrior) [inline packages/app/features/account/screen.tsx:339]:
```suggestion
          <Button.Icon theme="green_active" $theme-light={{col:"$color12"}}>{icon}</Button.Icon>
```

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]
Couple comments about colors. Let me know if you want to do a quick meaning to run through how radix colors work

File: packages/app/features/account/screen.tsx [hunk 2]
```diff
@@ -75,346 +72,292 @@ export function AccountScreen() {
...
-              color="$color12"
-              $theme-light={{ boc: '$color12' }}
-            >
-              Add
-            </Button.Text>
-          </XStack>
-        </LinkableButton>
-      ),
+      href: '/account/sendtag',
+      icon: <IconBadgeCheck size={22} color={'$primary'} $theme-light={{ color: '$olive' }} />,
```
[the remark this answers is not in the document]
Comment (stormcloud266) [reply packages/app/features/account/screen.tsx:90]: sounds good will push those changes shortly

File: packages/app/features/account/screen.tsx (hunk 1 above)
Comment (stormcloud266) [reply packages/app/features/account/screen.tsx]: Had to go with the Theme parent like you previous example

File: packages/app/features/account/screen.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (stormcloud266) [reply packages/app/features/account/screen.tsx]: okay they are all updated. noticed the avatar fallback was always olive and updated it to match the one in the nav

Screenshot 2024-09-13 at 2 26 43 PM

**Review by youngkidwarrior — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Sep 14, 2:43 AM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/711).

**Pull request merged**